config/cesm/config_files.xml

```diff
@@ -218,6 +218,7 @@
     <values>
       <value component="desp"      >$CIMEROOT/src/components/data_comps/desp</value>
       <value component="sesp"      >$CIMEROOT/src/components/stub_comps/sesp</value>
+      <value component="norcpm"    >$SRCROOT/components/norcpm</value>
```

Comment:
We normally list active components before data and stub. Please move this above line 219.